torchbenchmark/models/hf_MPT_7b_instruct/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+devices:
+  NVIDIA A100-SXM4-40GB:
+    eval_batch_size: 1
+eval_benchmark: false
+eval_deterministic: false
+eval_nograd: true
+not_implemented:
+- jit: true
```

Comment:
We have removed jit from the unit test cases, could you please remove the `not_implemented: - jit: true` part from the metadata file?

Reply:
Yes